src/sentry/notifications/notifications/rules.py

```diff
@@ -107,6 +111,11 @@ def get_context(self) -> MutableMapping[str, Any]:
             event=self.event,
             fallthrough_choice=self.fallthrough_choice,
         )
+        fallback_param = ""
+        # Piggybacking off of notification_reason that already determines if we're using the fallback
+        if notification_reason and self.fallthrough_choice == FallthroughChoiceType.ACTIVE_MEMBERS:
+            _, fallback_experiment = should_use_issue_alert_fallback(org=self.organization)
+            fallback_param = f"&ref_fallback={fallback_experiment}"
```

Comment:
@scttcper so we aren't recording`ref_fallback` right now on the FE. We do record the URL though but it might be easier to make this an explicit field for `issue_details.viewed` event.